components/resources/brave_components_strings.grd

```diff
@@ -422,6 +424,9 @@
       <message name="IDS_BRAVE_UI_REWARDS_OFF_TEXT2" desc="">without your consent.</message>
       <message name="IDS_BRAVE_UI_REWARDS_OFF_TEXT3" desc="">Today, Brave welcomes you to the new internet.</message>
       <message name="IDS_BRAVE_UI_REWARDS_OFF_TEXT4" desc="">One where your time is valued, your personal data is kept private, and you actually get paid for your attention.</message>
+      <message name="IDS_BRAVE_UI_REWARDS_OFF_TEXT_5" desc="">With your old browser, you paid to browse the web by viewing ads with your valuable attention. You spent your valuable time downloading invasive ad technology that transmitted your valuable private data to advertisers — without your consent.</message>
+      <message name="IDS_BRAVE_UI_REWARDS_OFF_TEXT_6" desc="">One where your time is valued, your personal data is kept private, and you actually get paid for your attention.</message>
+      <message name="IDS_BRAVE_UI_REWARDS_OFF_TITLE" desc="">Today, Brave welcomes you to the new internet.</message>
```

Comment:
`IDS_BRAVE_UI_REWARDS_OFF_TEXT3` is the same as `IDS_BRAVE_UI_REWARDS_OFF_TITLE` and `IDS_BRAVE_UI_REWARDS_OFF_TEXT4` is the same as `IDS_BRAVE_UI_REWARDS_OFF_TEXT_6` and `IDS_BRAVE_UI_REWARDS_OFF_TEXT1 ` is the same as `IDS_BRAVE_UI_REWARDS_OFF_TEXT_5`. We should only have one

Reply:
why can't we reuse existing keys in ui as well?

Reply:
I've removed `IDS_BRAVE_UI_REWARDS_OFF_TEXT_6` and `IDS_BRAVE_UI_REWARDS_OFF_TITLE`, and am now reusing existing translations.

For `IDS_BRAVE_UI_REWARDS_OFF_TEXT1` I've chosen to keep that separate as it isn't quite like `OFF_TEXT_5`, it has the remainder of it's text in `IDS_BRAVE_UI_REWARDS_OFF_TEXT2` for stylistic affect.

Reply:
can we change text in `IDS_BRAVE_UI_REWARDS_OFF_TEXT1`, because I don't see anywhere where text 1 would be used now, so it's a dead string

Reply:
removed